include/qpdf/Buffer.hh

```diff
@@ -46,6 +46,9 @@ class Buffer
     QPDF_DLL
     Buffer(std::string& content);
 
+    Buffer(Buffer const&) = delete;
+    Buffer& operator=(Buffer const&) = delete;
+
```

Comment:
I've added these to make it explicit that they are not supported instead of heaving compile errors about implicitly deleted constructors.

Reply:
Agreed.